opencog/cython/opencog/atomspace.pxd

```diff
@@ -88,6 +88,7 @@ cdef extern from "opencog/atomspace/Handle.h" namespace "opencog":
     cdef cppclass cHandle "opencog::Handle":
         cHandle()
         cHandle(UUID)
+        cHandle(void*)
```

Comment:
No, I was trying to say that the primary interface to cHandle should be cHandle(AtomPtr) instead of cHandle (UUID). You definitely must not have a void* interface, that will result in null pointer derefs and all sorts of other badness.